src/map/skill.cpp

```diff
@@ -16727,6 +16727,11 @@ int skill_unit_onplace_timer(struct skill_unit *unit, struct block_list *bl, t_t
 				skill_attack(skill_get_type(sg->skill_id), ss, &unit->bl, bl, sg->skill_id, sg->skill_lv, tick, flag);
 			}
 			break;
+
+		case UNT_GROUND_GRAVITATION:
+		case UNT_JACK_FROST_NOVA:
+			skill_attack(skill_get_type(sg->skill_id), ss, &unit->bl, bl, sg->skill_id, sg->skill_lv, tick, 0);
+			break;
```

Comment:
Please merge with the `// Units that deals simple attack` at the beginning of the switch.